apertium-kaz.kaz.rlx

```diff
@@ -256,25 +264,50 @@ SET NOT-ADV = WORDMARK - Adv ;
 
 SET S-BOUNDARY = CS | Interr | EOS ;
 
+# Universal Dependencies
+# ======================
+
+LIST @root = @root ;     # The root of the sentence, often a finite verb
+LIST @nsubj = @nsubj ;   # The nominal subject of the sentence
+LIST @amod = @amod ;       # 
+LIST @advmod = @advmod ; # An adverbial modifier
+LIST @case = @case ;     # The relation of an adposition to its head
+LIST @acl = @acl ;       # A clause which modifies a nominal
+LIST @nmod = @nmod ;     # Nominal modifier 
+LIST @dobj = @dobj ;     # The direct object of the sentence
+LIST @punct = @punct ;   # Any punctuation
+LIST @cop = @cop ;       # 
+LIST @nmod:poss = @nmod:poss ;
+LIST @obl = @obl ;
+LIST @obj = @obj ;
+LIST @advcl = @advcl ;
+LIST @aux = @aux ;
+LIST @parataxis = @parataxis ;
+LIST @det = @det ;
+LIST @csubj = @csubj ;
+LIST @nummod = @nummod ;
+LIST @dep = @dep ;       # Any remaining dependency
+
+
 # ======= #
-  SECTION # 
+  SECTION # repair 
 # ======= #
 
+
 REMOVE SUB:1 Err/Orth ; 
 
 REMOVE Err/Orth ; 
 
+SELECT (pass) IF (0 ("жоспарлан")) ;
```

Comment:
Don't use inline sets, e.g. ` ("жоспарлан") ` and `(pass)`. Make lists e.g.

```
LIST Pass = pass ;
LIST Жоспарлан = "жоспарлан"i ;
```

Reply:
Huh weird to do that for lemmas too.  What's the reasoning?

Reply:
Agreed... There are more like those, I'll iron them all out in one commit soon.

Reply:
@jonorthwash well, first of all,`i` is the `case-insensitive` flag, and it should be used here. And I think it is indeed a better practice to list lemmas on top of the file so that someone "porting" the .rlx file to another language can change them in a single place.